### ISSUE TITLE
Issue Regarding Survival Analysis

### ISSUE THREAD

**Issue by xianjieshen** [Starlitnightly/omicverse#68]
I'm encountering an issue regarding survival analysis. When I try to execute the following code:

Great to hear that you've followed the instructions to work with bulk data! If you have any further questions or encounter any issues along the way, feel free to ask for assistance. I'm here to help!

```
LIHC_tcga.survival_analysis('MYC', layer='deseq_normalize', plot=True, gene_threshold='median')
LIHC_tcga.adata
```
![240331010058](https://github.com/Starlitnightly/omicverse/assets/21005642/a193ad55-0b5c-44f2-a725-7957276b9b0b)

**Comment by Starlitnightly** [comment]
Hi, 

You can set the following code before `survial_analysis`

```python
from scipy import sparse
LIHC_tcga.adata.layers['deseq_normalize']=sparse.csr_matrix(LIHC_tcga.adata.layers['deseq_normalize'])
```

**Comment by xianjieshen** [comment]
![image](https://github.com/Starlitnightly/omicverse/assets/21005642/b8e2a748-36b9-4582-84ce-1b8b2a25bf68)
Thank you for your response! However, it seems that the problem still persists.

**Comment by Starlitnightly** [comment]
We have fixed this error in the version of `1.6.4`.
